### PR TITLE
Convert unstructured text event_logs argument field to jsonb.

### PR DESCRIPTION
**Pull request by efritz**
Convert type of `argument` in `event_logs` table to nullable jsonb.

@dan asked for this before merging https://github.com/sourcegraph/sourcegraph/pull/7988.

### CODE REVIEW

**Comment by dadlerj** [comment]
Actually, I was asking to blank out all of https://sourcegraph.com/search?q=+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+eventLogger.log%28:%5BeventName%5D%2C:%5Bargs%5D%29+&patternType=structural first!

We should start from scratch on these (not sure if these are all valid JSON, and even if so, we might not want to capture them).

**Comment by efritz** [comment]
@dadlerj is your suggested direction to still make it a jsonb column but just remove all existing argument data in the table?

**Comment by dadlerj** [comment]
Exactly. Deleting the existing argument data (which isn't even being logged today) means that we can start fresh. It means that once we DO start logging arguments (in https://github.com/sourcegraph/sourcegraph/pull/7988) there won't be any existing events that suddenly start throwing data in that we haven't audited in years.

**Comment by efritz** [comment]
(Possibly) blocked by https://github.com/sourcegraph/sourcegraph/pull/7995.

File: cmd/frontend/db/schema.md [hunk 1]
```diff
@@ -324,7 +324,7 @@ Referenced by:
  user_id           | integer                  | not null
  anonymous_user_id | text                     | not null
  source            | text                     | not null
- argument          | text                     | not null
+ argument          | jsonb                    |
```
Comment (tsenart) [inline cmd/frontend/db/schema.md]: Any objection to make this not null and have its default value be an empty object?

Comment (efritz) [reply cmd/frontend/db/schema.md]: No, I think that makes some things cleaner. Updating.

**Pull request merged**